i18n/es.toml

```diff
@@ -118,7 +118,63 @@ other = "Usuarios"
 [docs_label_i_am]
 other = "Yo soy..."
 
+# layouts > blog > pager
 
+[layouts_blog_pager_prev]
+other = "<< Anterior"
+
+[layouts_blog_pager_next]
+other = "Siguiente >>"
+
+# layouts > blog > list
+
+[layouts_case_studies_list_tell]
+other = "Cuéntanos tu historia"
+
+# layouts > docs > glossary
+
+[layouts_docs_glossary_description]
+other = "Este glosario tiene la intención de ser una lista completa y estandarizada de la terminología de Kubernetes. Incluye términos técnicos que son específicos de k8s, así como términos más generales que proporcionan un contexto."
+
+[layouts_docs_glossary_filter]
+other = "Filtrar terminos por categoría:"
+
+[layouts_docs_glossary_select_all]
+other = "Seleccionar todos"
+
+[layouts_docs_glossary_deselect_all]
+other = "Elmininar selección"
+
+[layouts_docs_glossary_aka]
+other = "Also known as"
+
+[layouts_docs_glossary_click_details_before]
+other = "Haz click en el símbolo"
+
+[layouts_docs_glossary_click_details_after]
+other = "para obtener información detallada sobre el término."
+
+# layouts > docs > search
+
+[layouts_docs_search_fetching]
+other = "Fetching results.."
```

Comment:
This is still in English. Maybe "Obtención de resultados"?

Reply:
Thanks! Yes, my bad. Commited before saving the file, fixed in the new squashed commit.